src/main/java/org/terasology/logic/health/RegenComponent.java

```diff
@@ -24,16 +11,35 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import static org.terasology.logic.health.RegenAuthoritySystem.BASE_REGEN;
-
+/**
+ * Not for direct access! Use regen events instead.
+ *
+ * @see org.terasology.logic.health.event.ActivateRegenEvent
+ * @see org.terasology.logic.health.event.DeactivateRegenEvent
```

Comment:
link? or is `@see` like link?

Reply:
I have no clue :shrug: nobody uses this anyways... :roll_eyes: